scripts/robotologyGitStatus.sh

```diff
@@ -27,7 +27,7 @@ printGitStatus () {
     (cd $1 && echo -n "${1}: " \
     && git rev-parse --abbrev-ref HEAD \
     && git log -1 --format="%cr|%s" \
-    && git status --short); \
+    && git status -sb --short); \
```

Comment:
It's already merged, but `-s` and `--short` are actually the same flag...

Reply:
But ``--short`` and ``-sb`` are not the same :grin:

Reply:
Feel free to fix!

Reply:
Here https://github.com/robotology/robotology-superbuild/pull/559